automated_analysis.py

```diff
@@ -601,7 +601,7 @@ def set_survey_percentages(survey_counts, total_survey_counts):
                     "Gender": gender_code.string_value,
                     "Number of Participants": demographic_counts[f"gender:{gender_code.string_value}"],
                     "Fraction of Relevant Participants": None if total_relevant_gender == 0 else
-                        demographic_counts[f"gender:{gender_code.string_value}"] / total_relevant_gender
+                    demographic_counts[f"gender:{gender_code.string_value}"] / total_relevant_gender
```

Comment:
I think this was easier to read with the indent

Reply:
updated thanks.